CONTRIBUTING.md

```diff
@@ -608,18 +608,34 @@ We use [Cypress](https://www.cypress.io/) for integration tests. Tests can be ru
 ```bash
 export SUPERSET_CONFIG=tests.superset_test_config
 export CYPRESS_BASE_URL="http://localhost:8081"
+# ENABLE_REACT_CRUD_VIEWS should be enabled for Cypress testing
+sed -i '/ENABLE_REACT_CRUD_VIEWS = False/c\ENABLE_REACT_CRUD_VIEWS = True' superset/config.py
+sed -i '/ENABLE_REACT_CRUD_VIEWS = os.environ.get("ENABLE_REACT_CRUD_VIEWS", False)/c\ENABLE_REACT_CRUD_VIEWS = True' tests/superset_test_config.py
```

Comment:
`sed` is super opaque to me, can we just alter `test/superset_test_config.py` to have `ENABLE_REACT_CRUD_VIEWS = True`, I think it's actually the default since recently too

Reply:
@mistercrunch I tried to se this TRUE but then python tests are failing:
```self = <tests.core_tests.TestCore testMethod=test_tablemodelview_list>
    def test_tablemodelview_list(self):
        self.login(username="admin")
        url = "/tablemodelview/list/"
        resp = self.get_resp(url)
        # assert that a table is listed
        table = db.session.query(SqlaTable).first()
>       assert table.name in resp
E       AssertionError: assert 'energy_usage' in '\n\n\n<!DOCTYPE html>\n\n\n\n\n<html>\n  <head>\n    <title>\n      \n        \n          Superset\n        \n      \...\n\n    \n  \n  \n  <!-- Bundle js crudViews START -->\n  \n  <!-- Bundle js crudViews END -->\n\n\n  </body>\n</html>'
E        +  where 'energy_usage' = energy_usage.name```

Now we can just pass it to the docker-compose up command